packages/runtime/src/state/react-page.ts

```diff
@@ -262,43 +209,20 @@ export function copyElementTree(
   return copyElementTreeNode(state, createReplacementContext(replacementContext))(copy)
 }
 
-function* traverseElementTree(
-  state: State,
-  elementTree: Documents.ElementData,
-): Generator<Documents.Element> {
-  yield elementTree
-
-  if (Documents.isElementReference(elementTree)) return
-
-  const descriptors = getComponentPropControllerDescriptors(state, elementTree.type)
-
-  if (descriptors == null) return
-
-  for (const [propKey, descriptor] of Object.entries(descriptors)) {
-    const children = Introspection.getElementChildren(descriptor, elementTree.props[propKey])
-
-    for (const child of children) {
-      if (!Documents.isElementReference(child)) yield* traverseElementTree(state, child)
-
-      yield child
```

Comment:
Note that if the child element was not a reference, we iterated over the same child twice: first, as a part of the recursive `traverseElementTree` call (the first `yield` above), and then explicitly here. This is [no longer the case](https://github.com/makeswift/makeswift/pull/894/files#diff-4c77c71de0cb4022cef56ea1671d57f758f297a229dab9ae8caf6f307c7b7918R104) in the new version: